config/validated_config_db_connector.py

```diff
@@ -12,6 +13,15 @@ def ValidatedConfigDBConnector(config_db_connector):
         config_db_connector.delete_table = validated_delete_table
     return config_db_connector
 
+def is_table_present_config_db(table):
```

Comment:
>is_table_present_config_db [](http://example.com/codeflow?start=4&length=26)

this implementation is not efficient.

Reply:
Made implementation more efficient in this PR- https://github.com/sonic-net/sonic-utilities/pull/2452

After PR 2452 is merged, I will rebase this MIRROR_SESSION PR to include those changes because this MIRROR_SESSION PR is blocked by YANG issue https://github.com/sonic-net/sonic-buildimage/issues/12397 , and PR 2452 will likely be merged first

Reply:
2452 is merged. Could you rebase this one?